pagopa/io-app
# chore(IT Wallet): [SIW-951] Add user payment methods in new wallet section

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5626 and https://github.com/pagopa/io-dev-api-server/pull/358


## Short description
This PR adds the user's payment methods in the new wallet section

## List of changes proposed in this pull request
- Added store and saga for the payments' `wallet` feature
- Added card mapping for payment methods
- Fixed a glitch in the ID Pay cards in wallet

## How to test
With the `io-dev-api-server`, make sure to checkout this branch: https://github.com/pagopa/io-dev-api-server/pull/358
From the **Profile** section, enable the **New wallet section** FF.
You should be able to see payment cards in the wallet.

## Preview
<img src="https://github.com/pagopa/io-app/assets/6160324/361f1808-48f9-425d-84d3-ef0ac7b6d62a" width="200" />
